contexts/edit.tsx

```diff
@@ -59,25 +60,9 @@ interface EditContextState {
   dynamicInstructions: string;
   setDynamicInstructions: React.Dispatch<React.SetStateAction<string>>;
   scriptPath: string;
-  droppedFiles: string[];
-  setDroppedFiles: React.Dispatch<React.SetStateAction<string[]>>;
-  droppedFileDetails: Map<
-    string,
-    {
-      fileName: string;
-      size: number;
-    }
-  >;
-  setDroppedFileDetails: React.Dispatch<
-    React.SetStateAction<
-      Map<
-        string,
-        {
-          fileName: string;
-          size: number;
-        }
-      >
-    >
+  droppedFiles: Map<string, FileDetail>;
+  setDroppedFiles: React.Dispatch<
```

Comment:
@ryanhopperlowe I DRYed the code bit, but it is not possible to move all the data convertions because setDroppedFiles is used in multiple places where update and delete happens with different implementation.